### ISSUE TITLE
is not null filter?

### ISSUE THREAD

**Issue by begyy** [collerek/ormar#115]
#django 
Model.objects.filter(category__isnull=True)

ormar ?
![image](https://user-images.githubusercontent.com/44405438/108848553-f0fc2280-7602-11eb-93a9-964455558ded.png)

**Comment by collerek** [comment]
Yeah there is no way to do this now, will add is null and is not null but right now I work on #74 as it's quite big (and in the meantime I have quite a lot of refactors that's why I need to finish it first)
